pkgs/development/php-packages/psysh/default.nix

```diff
@@ -19,7 +19,7 @@ let
   composerLock = fetchurl {
     name = "composer.lock";
     url = "https://github.com/bobthecow/psysh/releases/download/v${version}/composer-v${version}.lock";
-    hash = "sha256-ur6mzla3uXeFL6aEHAPdpxGdvcgzOgTLW/CKPbNqeCg=";
+    hash = "sha256-PQDWShzvTY8yF+OUPVJAV0HMx0/KnA03TDhZUM7ppXw=";
```

Comment:
cc @bobthecow did you force-push the 0.12.4 tag back then?

Reply:
Hmm on a second look it seems as if 0.12 never built, so perhaps the FOD hash was just wrong from the beginning.
Never mind then.